Apps/AccountDataAccess/src/Patient/PatientRepository.cs

```diff
@@ -146,7 +146,8 @@ public async Task BlockAccessAsync(BlockAccessCommand command, CancellationToken
 
             if (this.changeFeedEnabled)
             {
-                await this.messageSender.SendAsync(new[] { new MessageEnvelope(new DataSourcesBlockedEvent(command.Hdid, blockedAccess.DataSources), command.Hdid) }, ct);
+                IEnumerable<string> dataSourceValues = blockedAccess.DataSources.Select(ds => Enum.GetName(typeof(DataSource), ds))!;
```

Comment:
Enum.GetName could be simplified here with ds.ToString() or our EnumUtility.ToEnumString(ds).